packages/now-cli/test/dev/integration.js

```diff
@@ -290,8 +328,16 @@ function testFixtureStdio(directory, fn) {
       });
 
       await readyResolver;
-      const helperTestPath = (...args) => testPath(t, port, ...args);
-      await fn(t, port, helperTestPath);
+
+      const helperTestPath = async (...args) => {
+        if (!skipDeploy) {
+          await testPath(t, `https://${deploymentUrl}`, ...args);
```

Comment:
I wonder if doing this in a `Promise.all()` would make things much faster? 🤔

Reply:
I tried but somehow `Promise.all()` broke everything saying `t` is undefined and ran into 429 rate limit errors. So I'll leave it as is.